packages/client/tests/functional/issues/15084-mongo-logging/tests.ts

```diff
@@ -0,0 +1,55 @@
+import { NewPrismaClient } from '../../_utils/types'
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let newPrismaClient: NewPrismaClient<typeof PrismaClient>
+
+testMatrix.setupTestSuite(
+  () => {
+    let client: PrismaClient
+
+    test('should log queries', async () => {
+      client = newPrismaClient({
+        log: [
+          {
+            emit: 'event',
```

Comment:
Can we also the "normal" `stdout` logging somehow or is that to cumbersome?

Reply:
Sure added a snapshot for the stdout log.